### DISCUSSION TITLE
Addon-docs: Multiple Tab

### DISCUSSION THREAD

**Discussion by ChrTall** [storybookjs/storybook#25252] [Ideas]
### Is your feature request related to a problem? Please describe.

For design system documentation, we have a lot of content per component that is hard to be viewed in a single page format. In a high level, our doc is made up of 3 parts: Component Documentation, Component Guidelines, Pattern Guidelines(optional).

### Describe the solution you'd like

With the newest addon/doc that supports MDX, it would be perfect if I can make multiple tabs of mdx with custom title - each tab with its mdx.

### Are you able to assist to bring the feature to reality?

no

### Additional context

A good example of that: [Fluent UI React Docs](https://microsoft.github.io/fluent-ui-react/components/accordion/definition)

**Comment by ndelangen** [comment]
You should be able to create this yourself using https://storybook.js.org/docs/writing-docs/autodocs#customize-the-docs-container

**Comment by ChrTall** [comment]
@ndelangen 
I tried to reproduce the ExampleContainer from the documentation page, but I get the following error:

Property 'context' is missing in type '{ children: any; }' but required in type 'DocsContainerProps<Renderer>'.

Could you provide an example for a DocsContainer with 2 tabs?
